history/kadai-simplehistory-provider/src/main/java/io/kadai/simplehistory/impl/WorkbasketHistoryQueryImpl.java

```diff
@@ -517,7 +511,7 @@ public List<WorkbasketHistoryEvent> list(int offset, int limit) {
   @Override
   public List<String> listValues(
       WorkbasketHistoryQueryColumnName dbColumnName, SortDirection sortDirection) {
-    List<String> result = new ArrayList<>();
+    List<String> result;
```

Comment:
Why do we need to declare result at all? It's only returned if result = internalKadaiEngine.getSqlSession().selectList(LINK_TO_VALUE_MAPPER, this); is executed successfully.

Reply:
done